src/renderer/pages/player-page.js

```diff
@@ -1,6 +1,8 @@
 const React = require('react')
 const BitField = require('bitfield').default
 const prettyBytes = require('prettier-bytes')
+const path = require('path')
+const fs = require('fs')
```

Comment:
why are these required?

Reply:
Sorry about that, I was reviewing the possibility of using the torrents image in the `mediaSession`, but removed it after I encountered issues